sdks/go/test/integration/wordcount/wordcount_test.go

```diff
@@ -30,16 +32,20 @@ import (
 
 func TestWordCount(t *testing.T) {
 	tests := []struct {
-		lines []string
-		words int
-		hash  string
+		lines    []string
+		words    int
+		hash     string
+		name     string
+		counters int
```

Comment:
While it's need to parameterize these in the test, there's no value in it if they're all the same (1 counter, and we don't even check the value of it).

Instead, we have 2 different counters, the line-len distrtibution metric, and the small words counter. Put the wanted values of those counters here instead, rather than asserting the count (which is always 1, assuming the pipeline executed, as we found out). 

So we want to add fields so we can check the values of the counters.

```
		smallWordsCount int
		lineLenCount, linelenSum, lineLenMin, linLenMax int
```

And then we check those accordingly in the test loop.